rotors_gazebo_plugins/src/gazebo_imu_plugin.cpp

```diff
@@ -211,8 +260,18 @@ void GazeboImuPlugin::addNoise(Eigen::Vector3d* linear_acceleration,
 
 }
 
-// This gets called by the world update start event.
+
 void GazeboImuPlugin::OnUpdate(const common::UpdateInfo& _info) {
+
+  if(kPrintOnUpdates) {
+    gzdbg << __FUNCTION__ << "() called." << std::endl;
+  }
+
+  if(!pubs_and_subs_created_) {
+    CreatePubsAndSubs();
```

Comment:
can you document this somewhere? otherwise this knowledge is lost. maybe in the wiki?